packages/expo-sms/src/SMS.ts

```diff
@@ -1,21 +1,23 @@
-import { Platform } from '@unimodules/core';
-import ExpoSMS from './ExpoSMS';
+import { UnavailabilityError } from '@unimodules/core';
 
-type SMSResponse = {
-  result: 'unknown' | 'sent' | 'cancelled';
-};
+import ExpoSMS from './ExpoSMS';
+import { SMSResponse } from './SMS.types';
 
 export async function sendSMSAsync(
   addresses: string | string[],
   message: string
 ): Promise<SMSResponse> {
   const finalAddresses = Array.isArray(addresses) ? addresses : [addresses];
   if (!ExpoSMS.sendSMSAsync) {
-    throw new Error(`SMS.sendSMSAsync is not supported on ${Platform.OS}`);
+    throw new UnavailabilityError(`expo-sms`, `sendSMSAsync`);
```

Comment:
use single quotes for these, as we do elsewhere:
```suggestion
    throw new UnavailabilityError('expo-sms', 'sendSMSAsync');
```